packages/ketcher-react/src/script/ui/views/modal/Modal.tsx

```diff
@@ -0,0 +1,64 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import React, { useRef } from 'react'
+import useResizeObserver from 'use-resize-observer'
+import clsx from 'clsx'
+
+import modals from '../../dialog'
+import mediaSizes from './mediaSizes'
+import styles from './Modal.module.less'
+
+interface ModalProps {
+  modal: {
+    name: string
+    form: any
+    prop: any
+  }
+  onCancel: () => void
+  onOk: (result: any) => void
+}
```

Comment:
Please use Props and CallProps, see 
`ui/views/toolbars/ToolbarGroupItem/ToolbarGroupItem.tsx`
`ui/views/toolbars/TopToolbar/TopToolbar.tsx`
`ui/views/toolbars/TopToolbar/TopToolbar.container.tsx`
as example

Reply:
2179bc2582e70c743bc411e6970a6b5ba7b4f8c4